app/client/src/workers/validations.ts

```diff
@@ -359,6 +360,14 @@ export const VALIDATORS: Record<ValidationTypes, Validator> = {
           message: "This value is required",
         };
       }
+
+      if (value === "") {
+        return {
+          isValid: true,
+          parsed: config.params?.default || 0,
```

Comment:
```suggestion
          parsed: config.params?.default || "",
```
Wondering if we should default it to 0 or `""`, setting it to `0` would make it difficult to check if the value is set by the user?

Reply:
Same as above here.

Reply:
Tried this for the input as well:
![image](https://user-images.githubusercontent.com/1944800/131783201-11e4bc20-9ce7-4231-bcca-3da1daa09cce.png)

Reply:
Also noticed, after adding the widget `default text`, has an error state:
![image](https://user-images.githubusercontent.com/1944800/131798520-046a027c-50a6-4341-a76f-6c61a49d55f3.png)

Reply:
> Also noticed, after adding the widget `default text`, has an error state:
> ![image](https://user-images.githubusercontent.com/1944800/131798520-046a027c-50a6-4341-a76f-6c61a49d55f3.png)

This has been fixed on release have the pulled the latest release

Reply:
The change is for property pane fields which expect a number. When they have no value they get set to 0 in production.
Here we set it to zero but do not mark it as invalid so it won't be shown as an error


https://user-images.githubusercontent.com/67054171/132307472-57bf5e92-19b0-41be-a054-ed577d7d8aa8.mov